Add support for httpResource API

ivebeenlinuxed opened the pull request:
Provides httpResources for #359

cvetanstoimenov commented:
I extended a bit with allowing to send httpResourceOption as a parameter. 
PR: https://github.com/ivebeenlinuxed/ng-openapi-gen/pull/1

ivebeenlinuxed commented:
Hello @cvetanstoimenov! Thanks for the commit! Your changes look great!

I've squashed it and reverted the changes to package.json and package-lock.json as they didn't really contribute to the fix (We can let the maintainers increase the version number). I've then applied that commit to pull request here, and also merged it into my master branch. Hopefully you can see the git history is nice and clean this way.

I've also checked out your version into one of my latest projects, and everything seems to work a dream!

Hope that makes it easy for the maintainers to pull in.

bioc-druzgami commented:
Hello, are there any plans to merge this one? I hope it will become available soon 🤞

dguisinger commented:
I agree, I need this feature....

rhnatiuk commented:
Hello! Any plans on merging it?
